### ISSUE TITLE
Allow the renaming of files via the admin

### ISSUE THREAD

**Issue by Toutouwai** [processwire/processwire-requests#167]
### Short description of the enhancement

It is possible to rename images in the PW admin by clicking the image name in the thumbnail list view. It would be useful if something similar was possible for files.

### Why would the enhancement be useful to users?

Sometimes you need to rename a file after it has been uploaded. Currently this is not easy to do.

**Comment by gmclelland** [comment]
+1.  Here is a related issue https://github.com/processwire/processwire-requests/issues/49

**Comment by gmclelland** [comment]
For those that don't keep up in the forums, I just wanted to add there is also now a module https://processwire.com/talk/topic/19016-files-rename-replace/ for this created by @Toutouwai.

**Comment by cb2004** [comment]
@Toutouwai I agree, I recently had a client who knew how to do this with the images and I didn't have a good reason why they couldn't do it with files. I will give your module a whirl

@ryancramerdesign big +1 from me
